cmd/tfplugindocs/testdata/scripts/schema-json/migrate/time_provider_success_docs_website.txtar

```diff
@@ -48,6 +49,8 @@ cmpenv examples/ephemeral-resources/ephemeral_static/import_1.sh exp-examples/ep
 cmpenv examples/actions/example/example_1.tf exp-examples/actions/example/example_1.tf
 cmpenv examples/actions/example/example_2.tf exp-examples/actions/example/example_2.tf
 
+cmpenv examples/list-resources/example/example_1.tf exp-examples/list-resources/example/example_1.tf
```

Comment:
Should this be a `tfquery.hcl` file 👀 , wondering if the migrate command would extract example queries to `.tf` files or `.tfquery.hcl` files?